src/controllers/usersController.js

```diff
@@ -80,14 +78,14 @@ exports.loginUser = async(req, res) => {
                     const passwordMatch = bcrypt.compare(password, result.password);
                     if (passwordMatch) {
                         res.status(200).json({
-                            status: true,
+                            status: 'success',
                             message: "User Loggedin successfully",
                             password: result.password,
```

Comment:
Don't send the password back. Use JsonWebToken instead.

Reply:
Ok Noted